syz-verifier/main_test.go

```diff
@@ -479,14 +479,14 @@ func TestCreateReport(t *testing.T) {
 	got := string(createReport(&rr, 3))
 	want := "ERRNO mismatches found for program:\n\n" +
 		"[=] breaks_returns()\n" +
-		"\t↳ Pool: 1, Errno: 1, Flag: 1\n" +
-		"\t↳ Pool: 2, Errno: 1, Flag: 1\n\n" +
+		"\t↳ Pool: 1, Errno: 1 (operation not permitted), Flag: 1\n" +
+		"\t↳ Pool: 2, Errno: 1 (operation not permitted), Flag: 1\n\n" +
 		"[=] minimize$0(0x1, 0x1)\n" +
-		"\t↳ Pool: 1, Errno: 3, Flag: 3\n" +
-		"\t↳ Pool: 2, Errno: 3, Flag: 3\n\n" +
+		"\t↳ Pool: 1, Errno: 3 (no such process), Flag: 3\n" +
+		"\t↳ Pool: 2, Errno: 3 (no such process), Flag: 3\n\n" +
 		"[!] test$res0()\n" +
-		"\t↳ Pool: 1, Errno: 2, Flag: 7\n" +
-		"\t↳ Pool: 2, Errno: 5, Flag: 3\n\n"
+		"\t↳ Pool: 1, Errno: 2 (no such file or directory), Flag: 7\n" +
+		"\t↳ Pool: 2, Errno: 5 (input/output error), Flag: 3\n\n"
```

Comment:
I would now reorder Flag and Errno so that values align vertically (easier to compare visually):
```
		"\t↳ Pool: 1, Flag: 1, Errno: 2 (no such file or directory)\n" +
		"\t↳ Pool: 2, Flag: 1, Errno: 5 (input/output error)\n\n" +
```